lib/view/view_templates/tab_view_template.dart

```diff
@@ -0,0 +1,58 @@
+import 'package:flutter/material.dart';
+
+import '../../style.dart';
+
+class TabViewTemplate extends StatelessWidget {
+  final String title;
+  final List<String> tabs;
+  final List<Widget> children;
+  final List<Widget> actions;
+
+  final Color color;
+  final Color indicatorColor;
+
+  const TabViewTemplate(this.title,
+      {@required this.tabs,
+      @required this.children,
+      this.color = AppColors.green50,
+      this.actions,
+      this.indicatorColor});
```

Comment:
pls keep trailing comma for consistent formatting when function params span multiple lines